### ISSUE TITLE
Hi

### ISSUE THREAD

**Issue by marvoh** [IDCI-Consulting/ExtraStepBundle#1]
Looks like an interesting project. Where's the documentation?

**Comment by idciconsulting** [comment]
You're right, this is really useful, I do not have the time, but I've to do it !
If you need information, I'll help you with plaisure cause I want to improve the project.
And may be you could help me to do this documentation ?

Take a look to this projects too: 

- https://github.com/IDCI-Consulting/StepBundle
- https://github.com/IDCI-Consulting/ExtraFormBundle

Cause this is linked ;)
